### PR TITLE
learners: clarify features as future vs v3.5

### LINKED ISSUE #413: learner 3.5 capabilities section is invalid.
Any google search of learners takes you to this page which shows features in 3.5 that do not to my knowledge exist. I think we should probably change that section say possible future features or remove those that are not valid. The big ones are.

```
Make voting-member promotion fully automatic — Once a learner catches up to leader’s logs, a cluster can automatically promote the learner. etcd requires certain thresholds to be defined by the user, and once the requirements are satisfied, learner promotes itself to a voting member. From a user’s perspective, “member add” command would work the same way as today but with greater safety provided by learner feature.

Make learner standby failover node — A learner joins as a standby node, and gets automatically promoted when the cluster availability is affected.
```

https://etcd.io/docs/v3.3/learning/learner/#features-in-v35

### PR DESCRIPTION
**Pull request by hexfusion**
closes #413

### CODE REVIEW

File: content/en/docs/v3.3/learning/learner.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Learner only serves as a standby node until promoted: Leadership cannot be trans
 
 In addition, etcd limits the total number of learners that a cluster can have, and avoids overloading the leader with log replication. Learner never promotes itself. While etcd provides learner status information and safety checks, cluster operator must make the final decision whether to promote learner or not.
 
-## Features in v3.5
+## Proposed Features in Future Releases
```
Comment (chalin) [inline content/en/docs/v3.3/learning/learner.md:73]: I'd suggest using sentence-case for the heading (in this and other pages):

> ## Proposed features for future releases

Comment (hexfusion) [reply content/en/docs/v3.3/learning/learner.md:73]: makes sense

Comment (hexfusion) [reply content/en/docs/v3.3/learning/learner.md]: @chalin updated.

**Comment by hexfusion** [comment]
cc @gyuho @ptabor

**Review by chalin — APPROVED** [review]
LGTM (will wait on @gyuho and/or @ptabor to thumbs up)

**Review by ptabor — APPROVED** [review]

**Pull request merged**